sql/src/main/java/io/crate/operation/scalar/string/LowerFunction.java

```diff
@@ -100,20 +105,20 @@ public FunctionInfo info() {
     public Scalar<BytesRef, Object> compile(List<Symbol> arguments) {
         assert arguments.size() > 0 && arguments.size() < 3 : "invalid number of arguments";
 
-        if (arguments.size() < 2) {
-            currentLocale = Locale.getDefault();
-        } else {
+        Locale locale = Locale.getDefault();
```

Comment:
The getDefault call could also be done in an else case - minor perf improvement as getDefault accesses a volatile variable